websites/management/commands/update_content_metadata.py

```diff
@@ -22,7 +22,6 @@ def add_arguments(self, parser):
             "-t",
             "--type",
             dest="type",
-            default="resource",
```

Comment:
Add `required=True,` to avoid this:

```
Update None metadata for websites based on starter ocw-course, source=None, Press y for yes and anything else for no
```